Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM php:7.3-apache
+
+RUN apt-get update -y
```

Comment:
```
RUN apt-get update -y &&\
         apt-get install -y $dependencies
```

Using the `&& \` syntax will decrease image layers.

Reply:
Thank you for the tips. I'm already sending a new pull request.